src/components/home/HomePage.scss

```diff
@@ -0,0 +1,212 @@
+    }
+  }
+
+  .navbarLogo {
+    position: absolute;
+    max-width: 100px;
+    left: 100px;
+    cursor: pointer;
+  }
+
+  .navbarItemWrapper {
+    text-align: center;
+
+    a {
+      text-decoration: none;
+      color: white;
+    }
+  }
+
+  .iconTrio {
+    width: 25%;
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-left: 20px;
+    padding: 10px 5px 10px 5px;
+
+    .iconWrapper {
+      max-width: 35px;
+      margin-right: 15px;
+      min-width: 18px;
+    }
+
+    // FontAwesome classNames.
+    .fas,
+    .fab {
+      font-size: 22px;
+      cursor: pointer;
+      transition: font-size 0.3s ease-in-out;
+    }
+  }
+
+  $tabletFirst: 980px;
+  @media (max-width: $tabletFirst) {
+    .navbarItems .navLinkCol {
+      font-size: 95%;
+    }
+
+    .iconTrio {
+      .fas,
+      .fab {
+        font-size: 125%;
+      }
+    }
+  }
+
+  .hamburgerContainer {
+    display: none;
+  }
+}
+
+$breakToHamburger: 768px;
+@media (max-width: $breakToHamburger) {
+  .navbar {
+    position: absolute;
+    width: 200px;
+    height: 100vh;
+    min-height: 100vh;
+    right: 0;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background-color: transparent;
+    padding: 0;
+
+    .collapseMenuItems {
+      align-items: center;
+      background-color: #6676a0;
+      flex-direction: column;
+      justify-content: center;
+      height: 100%;
+      padding-left: 15px;
+      padding-right: 15px;
+      width: 200px;
+      transform: translateX(200px);
+      display: none;
+      transition: transform 1s linear;
+
+      &.hamburgerOpened {
+        display: flex;
+        transform: translateX(0px);
+        width: 100%;
+      }
+    }
+
+    .hamburgerContainer {
+      display: block;
+      width: 50px;
+      height: 50px;
+      position: absolute;
+      top: 10px;
+      right: 50px;
+      z-index: 2;
+      background-color: black;
+
+      .hamburgerIcon {
+        display: block;
+        width: 60%;
+        height: 3px;
+        position: absolute;
+        left: 50%;
+        top: 50%;
+        transform: translate(-50%, -50%);
+        background-color: white;
+        z-index: 2;
+
+        &::before,
+        &::after {
+          content: "";
+          height: 3px;
+          position: absolute;
+          background-color: white;
+          z-index: 2;
+        }
+
+        &::before {
+          top: -10px;
+          width: 50%;
+          right: 0;
+        }
+
+        &::after {
+          bottom: -10px;
+          width: 120%;
+          right: 0;
+        }
+      }
+    }
+
+    .navbarLogo {
+      bottom: 20px;
+    }
+
+    .navbarItems {
+      flex-direction: column;
+      width: 100%;
+      height: 40%;
+      order: 2;
+
+      .navLinkCol {
+        border-bottom: 0.1px solid #f5deb378;
+        transition: all 1.5s ease-in-out;
+
+        &:nth-of-type(1) {
```

Comment:
add a comment please. why is this nth-of-type here? imagine from my perspective that i'm reading this code. how am i supposed to know why the n-th of type has a special rule.